NEWS.md

```diff
@@ -1,3 +1,13 @@
+# pmforest 0.1.1
+
+## Bug Fixes:
```

Comment:
The colon made me pause, but that's consistent with the rest of the doc.  (Purely cosmetic of course, but good to keep it consistent.)

Reply:
Didn't realize we weren't doing that for the other packages, but yeah i'll definitely pay closer attention to that next time lol